packages/upgrade/package.json

```diff
@@ -34,6 +34,7 @@
     "@rehearsal/reporter": "^0.0.30",
     "@rehearsal/service": "^0.0.30",
     "@rehearsal/utils": "^0.0.30",
+    "@rehearsal/transforms": "^0.0.30",
```

Comment:
Could you place rename package to `@rehearsal/codefixes`, the transforms are only parts of a codefixes, the same as a hint :)